charts/fairspace-keycloak/values.yaml

```diff
@@ -20,13 +20,30 @@ fairspaceKeycloak:
         cpu: "500m"
         memory: "2048Mi"
   postgresql:
-    persistence:
-      enabled: true
-    resources:
-      # Request values are set by default in postgresql chart
-      limits:
-        cpu: "500m"
-        memory: "512Mi"
+
+keycloakx:
```

Comment:
the 'x' is not a typo? if not you can merge

Reply:
no, that's not a typo, but thanks for making sure!